src/NLog.Targets.Seq/LogEventInfoFormatter.cs

```diff
@@ -49,58 +51,87 @@ static LogEventInfoFormatter()
             };
         }
 
-        public static void ToCompactJson(IEnumerable<LogEventInfo> events, TextWriter output, IList<SeqPropertyItem> properties)
+        public static void ToCompactJson(IEnumerable<LogEventInfo> logEvents, TextWriter output, IList<SeqPropertyItem> properties)
         {
-            foreach (var loggingEvent in events)
+            foreach (var logEvent in logEvents)
             {
-                ToJson(loggingEvent, output, properties);
+                ToCompactJson(logEvent, output, properties);
                 output.Write(Environment.NewLine);
             }
         }
 
-        static void ToJson(LogEventInfo loggingEvent, TextWriter output, IList<SeqPropertyItem> properties)
+        public static void ToCompactJson(LogEventInfo logEvent, TextWriter output, IList<SeqPropertyItem> properties)
         {
-            if (loggingEvent == null) throw new ArgumentNullException(nameof(loggingEvent));
+            if (logEvent == null) throw new ArgumentNullException(nameof(logEvent));
             if (output == null) throw new ArgumentNullException(nameof(output));
             if (properties == null) throw new ArgumentNullException(nameof(properties));
 
+            // Without this call, logEventInfo.Properties behaves erratically when templates are used; stll trying
+            // to track down the cause (needs some debugging into NLog).
+            logEvent.FormattedMessage?.ToString();
```

Comment:
Yes, this is 4.5 :-) Brand new target for it!

The problem is that it's possible to access `LogEventInfo.Properties` without ever having triggered the internal calc method, thus the collection will be empty when instead it should have been populated from the template and params.

Shall I raise an issue on NLog for this one too? Cheers!